src/InteractsWithMessenger.php

```diff
@@ -52,4 +48,39 @@ final protected function messenger(?string $transport = null): TestTransport
 
         return $registry->get($transport);
     }
+
+    final protected function bus(?string $bus = null): TestBus
+    {
+        $this->init();
+        $container = self::getContainer();
+
+        if (!$container->has('zenstruck_messenger_test.bus_registry')) {
+            throw new \LogicException('Cannot access bus - is ZenstruckMessengerTestBundle enabled in your test environment?');
+        }
+
+        /** @var TestBusRegistry $registry */
+        $registry = $container->get('zenstruck_messenger_test.bus_registry');
+
+        return $registry->get($bus);
+    }
+
+    protected static function getContainer(): ContainerInterface
```

Comment:
Is it a good idea to extract it into a deprecated and internal trait and used here and in test? It will be removed once `self::$container` won't be supported anymore.

Reply:
This lib now requires php8 & symfony 5.4+, a rebase should remove the need for this.

Reply:
Rebased and removed